src/nifs.sol

```diff
@@ -0,0 +1,335 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "src/pasta/Pallas.sol";
+import "src/pasta/Vesta.sol";
+import "src/verifier/step2/Step2Data.sol";
+import "src/poseidon/Sponge.sol";
+
+library NIFSPallas {
+    uint256 constant private MOD = Pallas.P_MOD;
+    uint256 constant private NUM_FE_FOR_RO = 24; // <-- This seems strange, I'm only counting 15...
```

Comment:
Could this be out of date in Nova? Worth opening a Nova issue describing the count?

Reply:
This discrepancy has been resolved. The logic of how relaxed and non-relaxed IO vectors are absorbed in the RO is different, which I wasn't originally accounting for.